libc/math/fmax.c

```diff
@@ -0,0 +1,6 @@
+#include <math.h>
+
+int fmax(double x, double y)
```

Comment:
should return double as well, not int.
also lets just do float version and not double since we are on 32 bit.